registry.json

```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  },
+   {
+    "ticker": "PROOF",
+    "poolId": "pool17s9pcsnhgueer24p4lk53ruglr0r8009yxfrqvwgk5gx6r44cpe",
```

Comment:
This doesnt look like a real pool id.. I think it is actually `f40a1c4277473391aaa1afed488f88f8de33bde521923031c8b5106d` but please check and fix